sysbuild/Kconfig.zip

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+menuconfig DFU_ZIP
+	bool "Create DFU zip package"
+	depends on BOOTLOADER_MCUBOOT
```

Comment:
Why such dependency on MCUboot?
AFAIK the `dfu_application.zip` was also created when B0 was enabled (I think here: `nrf/subsys/bootloader/cmake/sign.cmake`) in child/parent.
Tested on nRF Desktop with `twister -v --build-only -T . -p nrf52kbd/nrf52832 --sub-test applications.nrf_desktop.zrelease.nrf52kbd` the zip file was not created.
Checked also in main using `west build --pristine -d build_kbd -b nrf52kbd -- -DCONF_FILE=prj_release.conf` and a dfu_application.zip was created.

Reply:
What's the point in having a zip if you can't do anything with it? The zip is for use with nrfutil or the mobile apps to load the files using MCUmgr. Without mcuboot you either have to design your own way of loading the updates or you have to flash an update using jlink

Reply:
Well, we can. In case of nRF Desktop application there is a tool `nrf/scripts/hid_configurator` which is used for the DFU also in case of B0.

CC: @MarekPieta